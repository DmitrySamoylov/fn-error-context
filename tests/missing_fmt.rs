use fn_error_context::error_context;

#[error_context]
fn do_stuff() -> anyhow::Result<()> {
    anyhow::bail!("error")
}

fn main() {}
