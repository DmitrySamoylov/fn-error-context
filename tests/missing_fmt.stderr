error: requires at least a format string argument
 --> $DIR/missing_fmt.rs:3:1
  |
3 | #[context]
  | ^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
